src/containers/App/App.tsx

```diff
@@ -1,6 +1,19 @@
+import { useAppDispatch } from "@/src/redux/hooks";
 import { Router } from "..";
+import { useEffect } from "react";
+import { getTokenFromLC } from "@/src/utils";
+import { fetchIsAuthUser } from "@/src/redux/slices";
 
 const App = () => {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
```

Comment:
This part will be executed on the fetch of /me data via request and response interceptors on the initial render